schema.graphql

```diff
@@ -314,6 +314,8 @@ type AssetTransaction @entity {
   # only applies to debt transfers
   fromAsset: Asset
   toAsset: Asset
+
+  realizedProfit: BigInt
```

Comment:
Would rename this to make it clear what methodology is used, and to support future other calculations
```suggestion
  realizedProfitFifo: BigInt
```